workflow/scripts/isoform-switch-analysis-init.R

```diff
@@ -0,0 +1,51 @@
+log <- file(snakemake@log[[1]], open="wt")
+sink(log)
+sink(log, type="message")
+
+library("tidyverse")
+library("sleuth")
```

Comment:
I can't see anything below that actually uses any sleuth functionality. Can we drop it here and from the environment `yaml` file?

Reply:
I think so.